test/format/format.h

```diff
@@ -189,14 +189,18 @@ typedef struct {
     WT_CONNECTION *trace_conn; /* Tracing operations */
     WT_SESSION *trace_session;
     WT_SPINLOCK trace_lock;
-    bool trace;
-    bool trace_bulk;
-    bool trace_cursor;
-    bool trace_mirror_fail;
-    bool trace_read;
+
+#define TRACE 0x01u
```

Comment:
Do you think we need `TRACE` flag? We could just check whether `trace_flags` is not zero instead?

Reply:
(just for the record, the answer to this is "yes" since it's used as a separate flag rather than indicating one of the other flags is set)